src/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,24 @@
+import { Fragment } from "react";
+import data from "../stretch-goal.json";
+import { Playlist } from "./Playlist.jsx";
+
+console.log("Playlists", data);
+
+export const Sidebar = () => {
+  return (
+    <Fragment>
+      <h2 id="Playlists">{data.message}</h2>
+      {data.playlists.items.map((list, i) => {
```

Comment:
For more clarity you could break this out. And it makes sense to call the variable something like element or item

```
const musicList = data.playlists.items
{musicList.map((song, i) => {
```